aries_cloudagent/protocols/present_proof/dif/pres_exch.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         ldp: Mapping = None,
         ldp_vc: Mapping = None,
         ldp_vp: Mapping = None,
+        di_vc: Mapping = None,
```

Comment:
i think we are using 'vc_di' not 'di_vc', to be consistent?

Reply:
I agree I took from this test vector from below repo and Ian confirmed that the presentation in the repository is right, and I'm not sure why it's di_vc instead of vc_di.

https://github.com/TimoGlastra/anoncreds-w3c-test-vectors/blob/main/test-vectors/dif-presentation-definition.json#L49

Reply:
It looks like that's what Credo is using (https://github.com/openwallet-foundation/credo-ts/blob/main/packages/core/src/modules/vc/models/ClaimFormat.ts)

Since we have to interoperate with Credo we should follow their conventions.  If it's wrong then we can always fix it later (we will need to sync changes with Credo).